p2p/swarm.go

```diff
@@ -29,6 +29,9 @@ import (
 // ConnectingTimeout is the timeout we wait when trying to connect a neighborhood
 const ConnectingTimeout = 20 * time.Second //todo: add to the config
 
+// UPNPRetries is the number of times to retry obtaining a port due to a UPnP failure
+const UPNPRetries = 20
```

Comment:
don't you think a more reasonable number would like - 5 ?

Reply:
Trying ports should be really fast (haven't benchmarked it, but it's milliseconds), so I don't think trying 20 is too much. The thing that takes time is discovering the gateway, and we only do that once.

Is there a different reason you think we should try less?

(I was originally gonna make it 200, but thought that the chance of 20 random ports failing because they're taken is slim, so made it that)

Reply:
ok my concern was that it would take a lot of time.